PSASSET-793 AWS Connection Key Rotation Policy Template

MitchellGerdisch opened the pull request:
### Description

Implements the AWS IAM user cloud credential key rotation policy.

### Issues Resolved

PSASSET-793

### Contribution Check List

- [X ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

cdwilhelm commented on compliance/cloud_credentials/aws/aws_connection_key_rotation_policy.pt:
```diff
@@ -0,0 +1,323 @@
+  run_script $js_aws_cloud_accounts, $ds_aws_clouds, $ds_cloud_accounts, $ds_account_tags, $param_rotation_period
+end
+
+# Find unattached volumes.
```
There is a number of areas that volumes are mentioned.  should be cleaned up to reference clouds instead.

MitchellGerdisch commented:
I removed volumes references and updated main README.
Also I changed the category from compliance to operational as per Bharat.
I'm holding on adding resolution bit until the "resolve_incident" function is available.
But the policy works as-is.

MitchellGerdisch commented:
Fixed the main README to be synched with master.
